app/views/seeds/index.rss.haml

```diff
@@ -16,8 +16,9 @@
             <p>GMO? #{seed.gmo}</p>
             <p>Heirloom? #{seed.heirloom}</p>
           - if seed.tradable?
-            :escaped
-              <p>Will trade #{seed.tradable_to} from #{seed.owner.location ? seed.owner.location : 'unknown location'}</p>
+            %p
```

Comment:
the haml will escape it. (as long as you don't put `:escaped` in front of it like we did)